languageserver/integration/commands.go

```diff
@@ -536,7 +537,7 @@ func (i *FlowIntegration) deployContract(conn protocol.Conn, args ...json.RawMes
 		return nil, errorWithMessage(conn, fmt.Sprintf("failed to load contract code: %s", path.Path), err)
 	}
 
-	_, deployError := i.sharedServices.Accounts.AddContract(signer, name, code, update)
+	_, deployError := i.sharedServices.Accounts.AddContract(signer, name, code, update, nil)
```

Comment:
Is this merely to match the API change? and no need to pass the arguments?
(If so, it shouldn't close https://github.com/onflow/flow-cli/issues/529?)

Reply:
This is to match the additional arguments used in addcontract function. This PR is a dependency of https://github.com/onflow/flow-cli/pull/589, which when both are merged will close https://github.com/onflow/flow-cli/issues/529